lib/sync/pull-request.js

```diff
@@ -9,12 +12,33 @@ const { getPullRequests: getPullRequestQuery } = require('./queries');
  * @returns {any} TBD
  */
 exports.getPullRequests = async (github, repository, cursor, perPage) => {
-  const { edges } = (await github.graphql(getPullRequestQuery, {
+  let edges;
+  const vars = {
     owner: repository.owner.login,
     repo: repository.name,
     per_page: perPage,
     cursor,
-  })).repository.pullRequests;
+  };
+
+  let asyncTags = [];
+  await statsd.asyncTimer(
+    async () => {
+      try {
+        ({ edges } = (await github.graphql(getPullRequestQuery, vars)).repository.pullRequests);
+        asyncTags.push('status:200');
+      } catch (error) {
+        if (error instanceof RequestError) {
+          asyncTags.push(`status:${error.status}`);
```

Comment:
It's pretty safe to say that we don't have many variance in the `status` right? I'm just thinking of the cardinality here.